### DISCUSSION TITLE
How to add a values which is in loop of each span

### DISCUSSION THREAD

**Discussion by kishordevakumar** [cypress-io/cypress#20863] [Questions and Help] [answered]
Hi All,
I'm new to cypress. I need your help to solve this

My scenario is like , there are tabs which will be dynamic based on data will appear. 
Inside tabs there is a value. Need to add those value and need to expect the added values is matching the value in a first tab which will be static.

**Comment by jjhelguero** [comment, marked as answer]
When you say a `tab` are you referring to a an element within a page or a browser tab?

**Reply by kishordevakumar** [reply]
within page

**Reply by jjhelguero** [reply]
Without seeing any of the DOM, you can use a `cy.get()` to get all your dynamic values, sum them up and then assert it equals the first 'tab'.

**Reply by kishordevakumar** [reply]
Since I'm new to cypress. Can you please give an example code. It will be appreciable

**Reply by kishordevakumar** [reply]
i can able to get the values. sum up only I'm struggling

**Reply by jjhelguero** [reply]
It would look something like this.
```
let sum = 0
cy.get('.dynamic-elements')
  .should('have.length.greaterThan', 1)
  .each( $item => {
    sum += Number($item.text())
  })
cy.get('.total')
  .then(Number)
  .should('eq', sum)
```

[Reference video](https://www.youtube.com/watch?v=51WPx7gvbAU)

**Reply by kishordevakumar** [reply]
Thank you so much. That's works for me!!
